modules/run/artifacts/opt/jboss/container/java/run/run-java.sh

```diff
@@ -3,6 +3,13 @@
 # Fail on a single failed command
 set -eo pipefail
 
+#These are defined explicitly here to avoid defining them in templates/jlink/Dockerfile
+export JBOSS_CONTAINER_UTIL_LOGGING_MODULE="/opt/jboss/container/util/logging"
+export JBOSS_CONTAINER_JAVA_RUN_MODULE="/opt/jboss/container/java/run"
+
+#This is moved here after deleting run-env.sh
+export JAVA_APP_DIR=/deployments
```

Comment:
here you unconditionally define JAVA_APP_DIR, which overwrites a user-provided value. The following tests check this behaviour and are failing in this PR: https://github.com/jboss-container-images/openjdk/blob/ubi9/tests/features/java/runtime.feature#L50